### PR TITLE
Sorted Travis build error

### PR DESCRIPTION
**Pull request by Zeus3101**
Sorts the CNNValue Import error. Had forgotten to include them in the parent `__init__.py` files.

### CODE REVIEW

**Comment by Het-Shah** [comment]
Why is Travis still failing?

**Comment by ajaysub110** [comment]
`atari-py` needs to be added to requirements, and a shape error needs to be resolved

**Comment by Het-Shah** [comment]
Theres an error in dqn.py in line number 193. Change it to self.env

**Comment by Zeus3101** [comment]
> Theres an error in dqn.py in line number 193. Change it to self.env

Yeah changed it, no idea why it's working although `env` is not defined

**Comment by Het-Shah** [comment]
still shape errors in `dqn.py`

**Comment by Sharad24** [comment]
@Zeus3101 Resolve the errors fast please? Also, make sure you run tests locally before.

It looks like some sort of endless loop somewhere.

File: genrl/deep/common/trainer.py [hunk 1]
```diff
@@ -214,9 +246,21 @@ def train(self):
...
+                    )
+                    self.buffer.push((
+                        phi_state, action, reward, phi_next_state, done
+                    ))
+                    phi_state = phi_next_state
+            else:
+                self.buffer.push((
+                    state, action, reward, next_state, done
+                ))
+                state = next_state
```
Comment (Het-Shah) [inline genrl/deep/common/trainer.py:263]: This won't work for DQN's mlp version now, change the if  condition to either `self.agent.__class__.__name__ == "DQN" and self.network_type == "cnn"` or add an else condition inside `self.agent.__class__.__name__ == "DQN"`

**Comment by Het-Shah** [comment]
Is it running locally now?

**Comment by Zeus3101** [comment]
It was but it was taking too long as I forgot to decrease the Trainer class' `steps_per_epoch`. It's working locally now.

**Pull request merged**